Makefile.PL

```diff
@@ -0,0 +1,81 @@
+#Makefile.PL version 1.0
+
+use strict;
+use warnings;
+use 5.010001;
+use ExtUtils::MakeMaker;
+
+my @scripts   = qw(htseq2multx.pl);
+my $fqmx_link = "https://github.com/brwnj/fastq-multx";
+my $fqmx_name = 'fastq-multx';
+
+WriteMakefile
+  (
+   NAME            => 'htseq2multx',
+
+   #Add these new keywords supported since 5.005
+   ($[ >= 5.005 ?
```

Comment:
Not sure how I missed this, but I get this warning:
```
$[ used in numeric ge (>=) (did you mean $] ?) at Makefile.PL line 17.
```

Looks like you meant [`$]` (`$OLD_PERL_VERSION` )](https://perldoc.perl.org/perlvar.html#%24OLD_PERL_VERSION)
```suggestion
   ($] >= 5.005 ?
```

Reply:
I did come across this alernate variable the other day.  This code was originally a copy/paste from a long time ago.  I suspected that maybe I should change it, but I haven't dug into it yet.

Reply:
It just seems that `$[` has been deprecated for a while (and perhaps isn't even functional anymore). Even if it was, it seems like it stores the "the index of the first element in an array, and of the first character in a substring".

[perlvar docs for `$[`](https://perldoc.perl.org/perlvar.html#%%5b)

I'm not really sure that changing the variable is really the best thing to do either, it seems from some looking that `use 5.005` would be better (though I wonder if we'd want to bump the version number up? I've only tested in `5.16.3`.

Reply:
Yeah.  I'll look into it.  Thanks for the info.

- [ ] Look into addressing usage of deprecated `$[`